addon/services/render-states.js

```diff
@@ -48,7 +48,8 @@ export default Service.extend(Evented, {
     this._super(...arguments);
     setProperties(this, {
```

Comment:
It might be better if we extract this and reuse in resetRenderState too.

Reply:
Have addressed in latest commit